docs/reference/deprecated.md

```diff
@@ -1,4 +1,4 @@
-Gutenberg's deprecation policy is intended to support backwards-compatibility for two minor releases, when possible. The current deprecations are listed below and are grouped by _the version at which they will be removed completely_. If your plugin depends on these behaviors, you must update to the recommended alternative before the noted version.
```

Comment:
Was this supposed to be removed without a replacement? It currently reads awkwardly as "intended to support compatibility for releases". I don't think it should have been removed.